### PR TITLE
[DO NOT MERGE] 3.4.0 API Auditing

### PR DESCRIPTION
**Pull request by xinrong-meng**
The PR shows API doc differences between 3.3.2 and the 3.4.0 RC4(the latest RC).

The PR isn't aimed to be merged, instead, it is for an easier review of API doc changes, as part of API Auditing for 3.4.0 release process.

### CODE REVIEW

File: site/docs/3.3.2/api/scala/org/apache/spark/sql/streaming/SinkProgress.html
Comment (HeartSaVioR) [inline site/docs/3.3.2/api/scala/org/apache/spark/sql/streaming/SinkProgress.html:1]: @HeartSaVioR check this out.

File: site/docs/3.3.2/api/scala/org/apache/spark/sql/streaming/StreamingQueryException.html
Comment (HeartSaVioR) [inline site/docs/3.3.2/api/scala/org/apache/spark/sql/streaming/StreamingQueryException.html:1]: @HeartSaVioR check this out.

File: site/docs/3.3.2/api/scala/org/apache/spark/sql/streaming/SinkProgress.html
Comment (HeartSaVioR) [reply site/docs/3.3.2/api/scala/org/apache/spark/sql/streaming/SinkProgress.html:1]: Looks OK.

File: site/docs/3.3.2/api/scala/org/apache/spark/sql/streaming/StreamingQueryException.html
Comment (HeartSaVioR) [reply site/docs/3.3.2/api/scala/org/apache/spark/sql/streaming/StreamingQueryException.html:1]: Looks OK as well.

File: site/docs/3.3.2/api/python/_modules/pyspark/sql/streaming/listener.html
Comment (HeartSaVioR) [inline site/docs/3.3.2/api/python/_modules/pyspark/sql/streaming/listener.html:1]: @HeartSaVioR check this out.

File: site/docs/3.3.2/api/python/_modules/pyspark/sql/streaming/query.html
Comment (HeartSaVioR) [inline site/docs/3.3.2/api/python/_modules/pyspark/sql/streaming/query.html:1]: @HeartSaVioR Check this out.

File: site/docs/3.3.2/api/python/_modules/pyspark/sql/streaming/readwriter.html
Comment (HeartSaVioR) [inline site/docs/3.3.2/api/python/_modules/pyspark/sql/streaming/readwriter.html:1]: @HeartSaVioR Check this out, especially why that's moved and renamed. Possibly matched with other files?

**Comment by dongjoon-hyun** [comment]
Could you close this, @xinrong-meng ?

**Comment by xinrong-meng** [comment]
Closed. Thanks @dongjoon-hyun for the reminder.

**Pull request closed without merging**